dev-libs/libpurple/libpurple-2.14.5.recipe

```diff
@@ -168,6 +168,8 @@ INSTALL()
 		$libDir/purple-2/*.la \
 		$libDir/lib*.la
 
+	ln -s $libDir/purple-2/libjabber.so.0 $libDir/libjabber.so.0
```

Comment:
atm libjabber.so links to libjabber.so.0.0.0, maybe symlink that one into $libDir? (not tested here)

Reply:
Just pushed― either's working fine on my system :)